src/components/App.jsx

```diff
@@ -158,12 +159,12 @@ export default function App() {
                 <rb.Offcanvas.Body>{nav}</rb.Offcanvas.Body>
               </rb.Navbar.Offcanvas>
               {currentWallet && (
-                <rb.Nav className="ms-sm-auto ms-lg-0 d-block order-sm-0">
-                  <Link to="/wallet" className="nav-link d-inline-block">
+                <rb.Nav className="ms-sm-auto ms-lg-0 d-inline-flex flex-row order-sm-0">
```

Comment:
Had to change this to `inline-flex` to vertically center the gear icon. I couldn't find another way but I feel like I just missed something obvious.